app/models/availability.rb

```diff
@@ -30,8 +30,4 @@ def people_to_pair
                 .where(fluency_level: 'target', language_id: native.language_id)
                 .where(user_id: target_users).order(:created_at)
   end
-
-  def self.user
-    User.find(user_id)
-  end
```

Comment:
Saw this wasn't being hit in test cov and don't believe it's being called anywhere so deleted. All tests still pass.

Reply:
Nice. I added that so that I wouldn't have to say User.find(user_id) in both other model methods but I guess it's just pulling user from relationships